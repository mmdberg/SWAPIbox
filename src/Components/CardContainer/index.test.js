import React from 'react';
import { shallow } from 'enzyme';
import { CardContainer } from './index';

describe('Card Container', () => {
  it('should match the snapshot', () => {
    let mockCards = [{ 
      name: 'Alderaan',
      climate: 'swamp'
    }];

    let wrapper = 
      shallow(<CardContainer cards={mockCards} favorites={mockCards}/>);
    expect(wrapper).toMatchSnapshot();
  });

});