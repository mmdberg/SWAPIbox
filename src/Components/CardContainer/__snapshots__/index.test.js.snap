// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Container should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    cards={
        Array [
            Object {
              "climate": "swamp",
              "name": "Alderaan",
            },
          ]
    }
    favorites={
        Array [
            Object {
              "climate": "swamp",
              "name": "Alderaan",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Unknown
          changeFavorites={undefined}
          className="card favorite"
          information={
                    Object {
                              "climate": "swamp",
                              "name": "Alderaan",
                            }
          }
/>,
      ],
      "className": "card-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {
          "changeFavorites": undefined,
          "className": "card favorite",
          "information": Object {
            "climate": "swamp",
            "name": "Alderaan",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Unknown
            changeFavorites={undefined}
            className="card favorite"
            information={
                        Object {
                                    "climate": "swamp",
                                    "name": "Alderaan",
                                  }
            }
/>,
        ],
        "className": "card-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {
            "changeFavorites": undefined,
            "className": "card favorite",
            "information": Object {
              "climate": "swamp",
              "name": "Alderaan",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
